### PR TITLE
Docs: Updating tags filter description

### PR DESCRIPTION
**Pull request by AnnuCode**
Changes: 
- updating `tags` filter description
- adding the sidebar menu link for navigation

### CODE REVIEW

**Review by i4ki — APPROVED** [review]
reads very well.
Thank you!

File: cli/orchestration/tag-filter.md [hunk 1]
```diff
@@ -11,30 +11,29 @@ The **Tag Filter** can be used in multiple Terramate features:
 - [stack.before](../stacks/configuration.md#stackbefore-setstringoptional)
 - `terramate <cmd> --tags <filter>`
 
-The filter returns a list of stacks containing `tags` that satisfy the filter
-query. The query language is best explained with some examples but a formal
+Using the `tags` filter, you can list or run commands on stacks based on tag conditions. Use `--tags` to filter stacks having specific tags and `--no-tags` to filter those without the specified tags. Both options support combining tag filters using AND (`:`) and OR (`,`) for more complex queries.
```
Comment (i4ki) [inline cli/orchestration/tag-filter.md:14]:
```suggestion
Using the `tags` filter, you can list or run commands on stacks based on tag conditions. Use `--tags` to filter stacks having specific tags and `--no-tags` to filter those without the specified tags. The `--tags` option support combining filters using AND (`:`) and OR (`,`) for more complex queries, whereas `--no-tags` should be used to specify the list of tags that must not be present in the stacks list and then only `,` is supported.
```

**Review by i4ki — CHANGES_REQUESTED** [review]
As [@luong-kumorebi](https://github.com/luong-komorebi) pointed out here https://github.com/terramate-io/terramate-docs/pull/93#issuecomment-2350963649

the `--no-tags` does not support the `:` (AND) operator.

This was my fault when I gave the examples, sorry.

File: cli/orchestration/tag-filter.md (hunk 1 above)
Comment (AnnuCode) [reply cli/orchestration/tag-filter.md:14]: @i4ki Thanks for the correction! 😄  I'm closing this PR as the other PR (#93) makes the necessary changes.

**Pull request closed without merging**